### PR TITLE
fixes for nix version in codespace

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

**Review by fl-y — APPROVED** [review]
Have no idea what I'm looking at but approving to speed the process up.

File: .github/workflows/release-artifacts.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     name: "Build runtime"
     runs-on:
       - self-hosted
-      - x64-cute
+      - x64-monster
```
Comment (blasrodri) [inline .github/workflows/release-artifacts.yml:20]: Why the change?

**Review by blasrodri — APPROVED** [review]
